.pre-commit-config.yaml

```
@@ -27,3 +27,7 @@ repos:
       verbose: true
       types: [text]
       require_serial: true
+- repo: https://github.com/gitleaks/gitleaks
+  rev: v8.16.1
```

Can we update it to latest version?
```suggestion
  rev: v8.18.0
```

---

updated, good catch!